samples/sensor/mhz19b/src/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <drivers/sensor.h>
+#include <drivers/sensor/mhz19b.h>
+
+void main(void)
+{
+	const struct device *dev;
+	struct sensor_value val;
+	int ret;
+
+	printk("Winsen MH-Z19B CO2 sensor application\n");
+
+	dev = DEVICE_DT_GET(DT_INST(0, winsen_mhz19b));
+	if (!dev) {
```

Comment:
You need to use device_is_ready() to check the device initialization status is OK.

Reply:
Done